data/engine-cli/docker_container_run.yaml

```diff
@@ -2331,6 +2331,7 @@ examples: |-
     | `--security-opt="seccomp=unconfined"`     | Turn off seccomp confinement for the container                                                                                                                                                                   |
     | `--security-opt="seccomp=builtin"`        | Use the default (built-in) seccomp profile for the container. This can be used to enable seccomp for a container running on a daemon with a custom default profile set, or with seccomp disabled ("unconfined"). |
     | `--security-opt="seccomp=profile.json"`   | White-listed syscalls seccomp Json file to be used as a seccomp filter                                                                                                                                           |
+    | `--security-opt="systempaths=unconfined"` | Turn off confinement for system paths (masked paths, read-only paths) for the container                                                                                                                           |
```

Comment:
Actually curious now why we chose to include the quotes in these examples (they seem redundant for these).

LOL, very unrelated, just curious if we had a reason for that 🤔